### PR TITLE
Update pytorch getting started test and setup examples tests to run in CI

### PR DESCRIPTION
**Pull request by oliverholworthy**
Depends on #65 
Follow-up to #47 

- Adding extension of notebook for pytorch example test to fix error
- Adding example tests to tox configuration
- Adds `./ci/test_unit.sh` script for container tests
- Updates getting started example notebooks to run on CPU and GPU
  - pytorch - handling pytorch device with conditional on `torch.cuda.is_available()`
  - tensorflow - update emebedding input dim to handle range of input values in sample data

### CODE REVIEW

File: examples/01b-Getting-started-Pytorch.ipynb [hunk 1]
```diff
@@ -335,7 +335,9 @@
     "optim = Adam\n",
```
Comment (bschifferer) [inline examples/01b-Getting-started-Pytorch.ipynb:335]: <p>Why do we have TensorFlow in this sentence?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='8'/>

Comment (bschifferer) [inline examples/01b-Getting-started-Pytorch.ipynb:335]: <p>Does the dataloader keep the dataset on CPU or GPU? How can we initialize the dataloader on CPU? Can we add a description and/or link to the documentation?</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/NVIDIA-Merlin/dataloader/pull/63/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='16'/>

Comment (oliverholworthy) [reply examples/01b-Getting-started-Pytorch.ipynb:335]: <p>Looks like a typo. Updated to PyTorch</p>

Comment (oliverholworthy) [reply examples/01b-Getting-started-Pytorch.ipynb:335]: <p>I've added a comment about this above this line. Can update for clarify. The default device to load into is determined based on availabliliy of GPU + if you have cudf installed. </p><p> </p><p>Which is  determined by the value of <code>merlin.core.dispatch.HAS_GPU</code>   The logic is here in the dataloader:</p><p> </p><p>https://github.com/NVIDIA-Merlin/dataloader/blob/v0.0.3/merlin/loader/loader_base.py#L78-L81 </p><p> </p><pre class="ql-syntax" spellcheck="false"># the dataloader determines whether or not to load data onto GPU or CPU
# depending on whether you have a GPU available and cudf installed
</pre>

**Comment by oliverholworthy** [comment]
The tensorflow example notebook is failing with a `StopIteration` running with tensorflow 2.11.

We can fix that with #65 by updating the dataloader to be handled by keras as an iterator instead of a sequence

**Review by bschifferer — APPROVED** [review]
